scripts/images/check_model_imports.py

```
@@ -52,12 +54,25 @@ def ensure_docker() -> None:
         raise RuntimeError("Docker is required but was not found on PATH.") from exc
 
 
+def _remove_image(image_reference: str) -> None:
+    """Remove a Docker image to free disk space."""
+    result = subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)